rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy

```diff
@@ -1459,7 +1461,35 @@ class ProjectController extends ControllerBase{
             }
             configProps.putAll(config)
         }
+        def currentProps = project.getProjectProperties() as Properties
+        def disableEx = currentProps.get("project.disable.executions")
+        def disableSched = currentProps.get("project.disable.schedule")
+        boolean isExecutionDisabledNow  = disableEx && disableEx == 'true'
+        boolean isScheduleDisabledNow = disableSched && disableSched == 'true'
+        def newExecutionDisabledStatus =
+                configProps[ScheduledExecutionService.CONF_PROJECT_DISABLE_EXECUTION] == 'true'
+        def newScheduleDisabledStatus =
+                configProps[ScheduledExecutionService.CONF_PROJECT_DISABLE_SCHEDULE] == 'true'
+
         def result=frameworkService.setFrameworkProjectConfig(project.name,configProps)
+
+        def reschedule = ((isExecutionDisabledNow != newExecutionDisabledStatus)
+                || (isScheduleDisabledNow != newScheduleDisabledStatus))
+        def active = (!newExecutionDisabledStatus && !newScheduleDisabledStatus)
+        if (reschedule) {
+            frameworkService.notifyProjectSchedulingChange(
+                    project,
+                    isExecutionDisabledNow,
+                    isScheduleDisabledNow,
+                    active
+            )
+            if (active) {
+                scheduledExecutionService.rescheduleJobs(frameworkService.isClusterModeEnabled() ? frameworkService.getServerUUID() : null, project)
+            } else {
+                scheduledExecutionService.unscheduleJobsForProject(project, frameworkService.isClusterModeEnabled() ? frameworkService.getServerUUID() : null)
+            }
+        }
+
```

Comment:
this duplicates the existing code in FrameworkController, i'd suggest refactoring both to have a new method in FrameworkService